src/mon/FSCommands.cc

```diff
@@ -202,9 +202,23 @@ class FsNewHandler : public FileSystemCommandHandler
     mon->osdmon()->propose_pending();
 
     // All checks passed, go ahead and create.
-    fsmap.create_filesystem(fs_name, metadata, data,
-                            mon->get_quorum_con_features());
+    auto fs = fsmap.create_filesystem(fs_name, metadata, data,
+        mon->get_quorum_con_features());
+
     ss << "new fs with metadata pool " << metadata << " and data pool " << data;
+
+    // assign a standby to rank 0 to avoid health warnings
+    std::string _name;
```

Comment:
Does it work to just call "maybe_promote_standby(fs)" here?  I see that calling tick() was problematic because it does a propose_pending(), but maybe_promote_standby in isolation should be alright

Reply:
We would actually use `maybe_resize_cluster` because there are no ranks `in`. I didn't want to use that becuase it needs to check the current epoch FSMap to see if a rank has become active: https://github.com/ceph/ceph/blob/eb5ca24f39025eae0525e5f0f9c8458f528bb46d/src/mon/MDSMonitor.cc#L1754-L1762

`maybe_resize_cluster` could be restructured to work but I'm not sure it's worth it.